index.html

```diff
@@ -38,6 +40,195 @@ <h1>
+                      <ul>
+                        <li class="option">Canopy</li>
+                        <li> <i class="fas fa-circle"></i>Back End Dev</li>
+                        <li><i class="fas fa-circle"></i>2015</li>
+                      </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups requires</p>
+                    <div class="languages inline">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                    </div>
+
+                    <a href="#" class="btn-one">See project</a>
+                  </li>
+                </ul>
+              </div>
+           </li>
+      </ul>
+</section>
+<section class="section-about">
+<div class="container">
+<div class="about-grid">
+    <div class="text">
+      <div class="headline-text">
+          <h1>About Myself</h1>
+
+            <p class="paragraph1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a
+             project you need coded, don’t hestiate to contact me.</p>
+             <p class="lets-connect"> LET'S CONNECT</p>
+
+             <ul class="buttons">
+                 <li><a> <img src="./images/twitter.png" alt=""> </a></li>
+                 <li><a> <img src="./images/Linkedin.png" alt=""></a></li>
+                 <li><a> <img src="./images/medium.png" alt=""></a></li>
+                 <li><a> <img src="./images/github.png" alt=""></a></li>
+                 <li><a> <img src="./images/angellist.png" alt=""></a></li>
+             </ul>
+          <br>
+          <a href="#" class="btn-one">Get My Resume</a>
+          </div>
+       </div>
+
+    <div class="text">
+    <ul class="list">
+        <li class="languages flex">
+            
+            <a href="#"> <h2 class="mark-up">Languages</h2></a>
+            <ul class="programminglang">
+            <li>
+                
+                <img src="images/jscript.png" alt="javascript">
+                <pre>JavaScript</pre>
+                
+            </li>
+            <li>
+                
+                <img src="images/html.png" alt="hyper text mark-up Language">
+                <pre>HTML</pre>
+                
+            </li>
+                
+            <li class="lang1">
+                
+                <img src="images/css.png" alt="Cascading style sheet">
+                <pre>CSS</pre>
+                
+            </li>
+                
+            
+          </ul>
+        </li>
+        
+        <li>
+            <a href="#"> <h2 class="mark-up"> Frameworks</h2> </a>
+        </li>
+        
+        <li>
+            
+          <a href="#"> <h2 class="mark-up">Skills</h2> </a>
+  
+        </li>
+    </ul>
```

Comment:
- [ ] Fix margin/padding that you can see in yellow and blue in the image below.
- [ ] Add right and down arrows as `img` tag in after Languages and project technologies in this section. 

![image](https://user-images.githubusercontent.com/60164239/129462027-740c14a7-bdc6-4292-b215-3f14ecff95d9.png)